conference.js

```diff
@@ -1313,11 +1313,39 @@ export default {
         }
     },
 
+    async joinRoom(roomName) {
+        this.roomName = roomName;
+
+        const initialOptions = { startScreenSharing: false };
+
+        const { tryCreateLocalTracks, errors } = this.createInitialLocalTracks(initialOptions);
+        const localTracks = await tryCreateLocalTracks;
+
+        this._displayErrorsForCreateInitialLocalTracks(errors);
+        localTracks.forEach(track => {
+            if ((track.isAudioTrack() && this.isLocalAudioMuted())
+                || (track.isVideoTrack() && this.isLocalVideoMuted())) {
+                track.mute();
+            }
+        });
+        this._createRoom(localTracks);
+
+        return new Promise((resolve, reject) => {
+            (new ConferenceConnector(resolve, reject)).connect();
+        });
+    },
+
     _createRoom(localTracks) {
+        const conferenceOptions = this._getConferenceOptions();
+        const customDomain = /_[-\da-f]{36}$/.test(APP.conference.roomName)
+            ? `breakout.${conferenceOptions.hosts.domain}`
```

Comment:
Agree that this would be good to do. Same would apply to jicofo and prosody. I would preferably do this in a separate PR introducing a customisation feature for the breakout rooms domain. Would that make sense?

Reply:
BTW: Domain handling logic has been moved to [createRoom in lib-jitsi-meet](https://github.com/jitsi/lib-jitsi-meet/pull/1654/files#diff-d86e76d490765474507e19efc80ad0127d74bad1b3dc31a05a4880838fd2b311). This is a more central place than in the client app and does not require using the customDomain workaround.

Reply:
Removed extra muc component definition for breakout rooms. They are using the main muc domain now. Breakout rooms are distinguished by a #breakout_uuid suffix.